internal/chain/chainsubscriber.go

```diff
@@ -19,7 +19,10 @@ import (
 const (
 	// updatesBatchSize is the maximum number of updates to fetch in a single
 	// call to the chain manager when we request updates since a given index.
-	updatesBatchSize = 50
+	updatesBatchSize = 1e3
```

Comment:
1000 seems like a lot. That's 1000MB of ram constantly in use during syncing. Unless it's that much slower I'd go for 100.